src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/jpa/entity/UnallocatedCaseEntity.kt

```diff
@@ -34,5 +34,8 @@ data class UnallocatedCaseEntity(
   val initial_appointment: LocalDate? = null,
 
   @Column
-  val status: String
+  val status: String,
+
+  @Column
+  val previous_conviction_date: LocalDate? = null
```

Comment:
Not sure on why we use the null initialisation for entity properties. Can we just declare  LocalDate?